tags/TagMap.md

```diff
@@ -58,8 +59,20 @@ For now, valid values of `TagTTL` are
 
 ## Tag Conflict Resolution
 If a new tag conflicts with an existing tag then the new tag takes precedence. Entire `Tag` along 
-with `TagValue` and `TagMetadata` is overwritten with a most recent tag (either it is locally
-generated or received fromi a remote peer).
+with `TagValue` and `TagMetadata` is replaced by the most recent tag (either it is locally
```

Comment:
nit: I would replace 'either' with 'whether' or 'regardless of'

Reply:
fixed.